src/shared/components/donate/donate.js

```diff
@@ -22,7 +22,7 @@ class Donate extends Component {
             enter the tech industry, and code the future.
           </p>
           <p>Thank you for supporting our mission!</p>
-          <LinkButton text="Donate Now" link="https://donorbox.org/operationcode" />
+          <LinkButton text="Donate Now" link="https://donorbox.org/operationcode" target="_blank" rel="noopener noreferrer" />
```

Comment:
Some link internally, and I didn't spend enough time on why the `isExternal` option wasn't working.